File: tests/xcalablemp/global-view/align/C-Round/offset.c
#include <stdio.h>
#include <xmp.h>
#pragma xmp nodes p(*)
#pragma xmp template t(2:11)
#pragma xmp distribute t(block) onto p
int a[10];
#pragma xmp align a[i] with t(i+2)

int main(void){
#pragma xmp loop on t(i)
  for(int i=2;i<10;i++)
    a[i] = i;
  
#pragma xmp task on p(1)
  {
    printf("PASS\n");
  }
  return 0;
}

File: tests/XMP/global-view/align/F/offset.f90
program main
  include 'xmp_lib.h'
!$xmp nodes p(*)
!$xmp template t(3:12)
!$xmp distribute t(block) onto p
integer a(10)
!$xmp align a(i) with t(i+2)

!$xmp loop on t(i+2)
do i = 3, 10
   a(i) = i
end do

!$xmp task on p(1)
  write(*,*) "PASS"
!$xmp end task
  
end program main
